src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -99,9 +99,20 @@ protected function extractAttributes($object, $format = null, array $context = [
     {
         $reflectionObject = new \ReflectionObject($object);
         $attributes = [];
+        $checkPropertyInitialization = \PHP_VERSION_ID >= 70400;
 
         do {
             foreach ($reflectionObject->getProperties() as $property) {
+                if ($checkPropertyInitialization) {
+                    if (!$property->isPublic()) {
+                        $property->setAccessible(true);
```

Comment:
why is this call needed?

Reply:
I think, this is what @vudaltsov meant when he wrote

> It appears that `ReflectionProperty::isInitialized` can be called only on accessible properties